assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -4772,253 +4772,6 @@ tests:
               relation: owner
             expectation:
               - repo:1
-      - model: | #complex model
```

Comment:
Fun fact, this sub-stage within this test fails currently on `main` if you comment out the two stages above it. We have something weird going on with our multi-stage tests in `tests/check` and `tests/listobjects`. I didn't have time to investigate the full issue.

That being said, this test is actually invalid, because when we run this test with passing all of these tuples as contextual tuples, the number of tuples here well exceeds the 20 tuple limit for contextual tuples (see code links below). As a result of this, I decided to remove this test from the suite. It may be pertinent to add it as just a regular test somewhere else, but it doesn't belong here because it's invalid.

https://github.com/openfga/openfga/blob/main/tests/listobjects/listobjects.go#L105

https://github.com/openfga/openfga/blob/main/tests/listobjects/listobjects.go#L177

Reply:
The test is not invalid, please do not remove it. It failed when you commented out the previous stages because that triggers the contextualTuple harness, which fails in this case because we didn't account for the 20 tuple limit that this test in particular exceeds. Please see the fix in https://github.com/openfga/openfga/pull/1336

Reply:
If I'm reading correctly, this is just an extra change thrown into the PR. Invalid or not, I'd prefer we keep limit the scope of this PR to the check resolver delegation.

Reply:
I agree, but i add that this test is valid, if it failed in the scope of building this PR i'd like to know that

Reply:
I'm not observing the test failure. I say we should revert for this PR.

Reply:
Something odd is going on, I'll investigate further. I'll revert the change. We should follow up on this though, because it is an invalid test. Maybe validation or something else is not working somewhere in the call stack.